helm/values-uat.yaml

```diff
@@ -53,8 +53,11 @@ microservice-chart:
       WARMUP_REQUEST_NEW_TRANSACTION_NOTICE_CODE_PREFIX: "3020"
       LOGO_CARD_BRANDING_MAPPING: >
         { VISA:'https://dev.checkout.pagopa.it/assets/creditcard/carta_visa.png',  MASTERCARD:'https://dev.checkout.pagopa.it/assets/creditcard/carta_mc.png', DINERS:'https://dev.checkout.pagopa.it/assets/creditcard/carta_diners.png', MAESTRO:'https://dev.checkout.pagopa.it/assets/creditcard/carta_maestro.png', AMEX:'https://dev.checkout.pagopa.it/assets/creditcard/carta_amex.png', UNKNOWN:'https://dev.checkout.pagopa.it/assets/creditcard/generic.png' }
-
       SERVER_ERROR_INCLUDE_STACKTRACE: "never"
+      ELASTIC_APM_SERVICE_NAME: pagopa-ecommerce-transactions-service
```

Comment:
How about set blue service name with '-blue' suffix in order to distinguish UAT logs for blue and green envs?